libraries/AP_Soaring/POMDP_Soar.cpp

```diff
@@ -0,0 +1,634 @@
+    // @Units:
+    // @Range: 0 1000
+    // @User: Advanced
+    AP_GROUPINFO("LOOP", 6, POMDSoarAlgorithm, pomdp_loop_load, 1),
+
+    // @Param: ROLL1
+    // @DisplayName: POMDP's maximum commanded roll angle.
+    // @Description: Maximum commanded roll angle in the POMDP used by POMDSoar.
+    // @Units: degrees
+    // @Range: 0 1000
+    // @User: Advanced
+    AP_GROUPINFO("ROLL1", 7, POMDSoarAlgorithm, pomdp_roll1, 15),
+
+    // @Param: ROLL2
+    // @DisplayName: POMDP's minimum commanded roll angle.
+    // @Description: Minimum commanded roll angle in the POMDP used by POMDSoar.
+    // @Units: degrees
+    // @Range: 0 1000
+    // @User: Advanced
+    AP_GROUPINFO("ROLL2", 8, POMDSoarAlgorithm, pomdp_roll2, 45),
+
+    // @Param: RRATE
+    // @DisplayName: The sailplane UAV's roll rate increment used by POMDSoar
+    // @Description: The sailplane UAV's roll rate increment used by POMDSoar.
+    // @Units: degrees/second
+    // @Range: 0 1000
+    // @User: Advanced
+    AP_GROUPINFO("RRATE", 9, POMDSoarAlgorithm, pomdp_roll_rate, 75),
+
+    // @Param: N_ACT
+    // @DisplayName: POMDP number of actions
+    // @Description: Number of actions in the POMDP used by POMDSoar. The roll angle input commands corresponding to actions are endpoints of (N_ACT-1) equal intervals between ROLL2 and ROLL1 (inclusive).
+    // @Units: seconds
+    // @Range: 1 254
+    // @User: Advanced
+    AP_GROUPINFO("N_ACT", 10, POMDSoarAlgorithm, pomdp_n_actions, 2),
+
+    // @Param: I_MOMENT
+    // @DisplayName: I-moment coefficient
+    // @Description: Airframe-specific I-moment coefficient used by POMDSoar to model the trajectory corresponding to a given commanded roll angle.
+    // @Units:
+    // @Range: -10000 10000
+    // @User: Advanced
+    AP_GROUPINFO("I_MOM", 11, POMDSoarAlgorithm, I_moment, 0.00257482),
+
+    // @Param: K_AILERON
+    // @DisplayName: Aileron K coefficient
+    // @Description: Airframe-specific aileron K coefficient used by POMDSoar to model the trajectory corresponding to a given commanded roll angle.
+    // @Units: seconds
+    // @Range: -10000 10000
+    // @User: Advanced
+    AP_GROUPINFO("K_AIL", 12, POMDSoarAlgorithm, k_aileron, 1.44833047),
+
+    // @Param: K_ROLLDAMP
+    // @DisplayName: Roll dampening K coefficient
+    // @Description: Airframe-specific roll-dampening K coefficient used by POMDSoar to model the trajectory corresponding to a given commanded roll angle.
+    // @Units:
+    // @Range: -10000 10000
+    // @User: Advanced
+    AP_GROUPINFO("RLLDMP", 13, POMDSoarAlgorithm, k_roll_damping, 0.41073589),
+
+    // @Param: ROLL_CLP
+    // @DisplayName: CLP coefficient
+    // @Description: Airframe-specific CLP coefficient used by POMDSoar to model the trajectory corresponding to a given commanded roll angle.
+    // @Units:
+    // @Range: -10000 10000
+    // @User: Advanced
+    AP_GROUPINFO("RLLCLP", 14, POMDSoarAlgorithm, c_lp, -1.12808702679),
```

Comment:
the @ param again.. I see more like this too. please check them all

Reply:
Should be fixed now.